engine.py

```diff
@@ -199,7 +231,7 @@ def __str__(self) -> str:
 
 
 class GoogleSpeechToTextEngine(Engine):
-    def __init__(self, cache_extension: str = 'ggl', model: str = None):
+    def __init__(self, cache_extension: str = '.ggl', model: str = None):
         self._client = speech.SpeechClient()
 
         self._config = speech.RecognitionConfig(
```

Comment:
not sure if it's because I have the latest version of Google cloud speech and this changed, but this `RecognitionConfig` class seems to take a dictionary, not an argument list any longer. At least, that's what my IDE is saying, I don't have Google credentials to run in. Can you upgrade yours and check if you get the same warning?